src/jlapi.c

```diff
@@ -318,6 +318,26 @@ JL_DLLEXPORT jl_value_t *(jl_typeof)(jl_value_t *v)
     return jl_typeof(v);
 }
 
+JL_DLLEXPORT int8_t (jl_gc_managed_enter)(void)
+{
+    return jl_gc_managed_enter();
+}
+
+JL_DLLEXPORT void (jl_gc_managed_leave)(int8_t state)
+{
+    jl_gc_managed_leave(state);
+}
+
+JL_DLLEXPORT int8_t (jl_gc_unmanaged_enter)(void)
```

Comment:
I would rather call this `gc_safe` --- you call it that in your comments elsewhere, and I think it's clearer.